src/node.cc

```diff
@@ -429,7 +429,7 @@ void ResetSignalHandlers() {
 #endif  // __POSIX__
 }
 
-static std::atomic<uint64_t> init_process_flags = 0;
+static std::atomic<uint32_t> init_process_flags = 0;
```

Comment:
Do you want to leave a comment on https://github.com/nodejs/node/blob/0a3d0e0ac350820cb52d0c61a73e2fc7cdc09737/src/node.h#L231 to also change that to a uint32_t in a future major version, to make sure that the two value widths here are aligned?

Reply:
Changed.
Now if uint32_t is lock-free, this line is not needed ?
https://github.com/nodejs/node/blob/0a3d0e0ac350820cb52d0c61a73e2fc7cdc09737/src/node.cc#L437

Reply:
That line is definitely needed! It’s specifically for catching things like #45152, so that we know if this expectation is violated for a given platform.